tox.ini

```diff
@@ -18,6 +24,9 @@ deps =
     dj16: -rtest_requirements/django-1.6.txt
     dj17: -rtest_requirements/django-1.7.txt
     dj18: -rtest_requirements/django-1.8.txt
+    ; TODO: replasce after new version release on pypi (with py3 support)
+    py34: https://github.com/aldryn/aldryn-people/archive/master.zip
+    py33: https://github.com/aldryn/aldryn-people/archive/master.zip
```

Comment:
I'd prefer exact commit SHA ids here instead of 'master'. Otherwise the day the code will silently change tests will start to fail.